internal/output/output.go

```diff
@@ -0,0 +1,57 @@
+package output
+
+const (
+	NameKey       = "__key"
+	NameKeyType   = "__key_type"
```

Comment:
Possible typo `__key.type`? If it intentional, ignore this comment.

Reply:
That's intentional.